CMakeLists.txt

```diff
@@ -1,30 +1,27 @@
 cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
Is it possible to format it in such way so only your changes are introduced ? Its really hard to see what are your changes from the original ones